physcraper/local_blast.py

```diff
@@ -24,10 +24,9 @@ def del_blastfiles(workdir):
     """Deletes all files in the local blast folder.
     """
     try:    
-	shutil.rmtree(os.path.join(workdir, "blast"))
+	      shutil.rmtree(os.path.join(workdir, "blast"))
     except: 
-	sys.stderr.write("Blast folder was not removed. Maybe it was not present?")
-
+        sys.stderr.write("Blast folder was not removed. Maybe it was not present?")
```

Comment:
That does not seem to me like the  correct fix. I would suggest changing your editor setting to show the difference between tabs and space, you have both tabs and space in these lines, and hence why you have the wrong indentation. You need to get rid of tabs.